Docs.Site/Docs/Reference/FAQ.markdown

```diff
@@ -18,6 +18,17 @@ See the [Creating and publishing a package](http://docs.nuget.org/docs/creating-
 
 In order to publish them onto your MyGet feeds, you'll need to create a MyGet account first.
 
+## When do packages appear on the feed after pushing them successfully?
+
+We often get this question: 
+"I pushed a package successfully but it is not yet available (even though activity log shows it). Why?"
+
+Usually, your packages will appear on the feed within a minute. Depending on load and other factors, this could take up to 5 minutes. Very sporadically even longer (this can happen for instance when we are scaling out during burst load scenarios).
+
+Note that MyGet.org is an eventual consistent system (and so is NuGet.org).
```

Comment:
I think this should be either ```eventually consistent``` or ```eventual consistency```.  What you have just now is a bit of a mix of the two.  Just a small suggestion.